### ISSUE TITLE
Loading UTF-8 localized property files

### ISSUE THREAD

**Issue by noncom** [michaelklishin/propertied#6]
Hello, in my project I am loading localized property files which are in UTF-8, and the default implementation of `Properties.load()` does not support UTF-8. The workaround is to create a custom `InputStreamReader` with the encoding enabled. As this misfeature renders the otherwise wonderful propertied library unusable for such cases, I suggest to think of some kind of a fix for it.

Here's my java pseudocode example for loading UTF-8 properties:

```
    InputStream is = // get an input stream from file
    Properties res = new Properties();
    try {
        res.load(new InputStreamReader(is, "UTF-8"));
    } catch (IOException e) {
        e.printStackTrace();
    } finally {
        try {
            is.close();
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
```

**Comment by michaelklishin** [comment]
Feel free to submit a PR. I believe assuming that the files are UTF-8 encoded by default (but ideally having an option to use any encoding) is very reasonable in this day and age.

**Comment by noncom** [comment]
Okay, I'll make a PR soon. Yes, providing an option to specify the encoding would be better and truer than to rely on it being a specific one.. I'll look into it.

**Comment by aisamu** [comment]
The `java.util.Properties` docs mention that `load(InputStream)` assumes an ISO 8859-1 character encoding. I've had the same issue you mention and replacing the InputStream with a Reader (which uses the system default encoding) fixed it. If there aren't hidden issues with this approach, I'd gladly submit the PR. 

```
(extend-protocol PropertyReader
[...]
java.io.File
(load-from [input]
  (with-open [rdr (io/reader input)]
     (doto (Properties.)
       (.load rdr))))
[...]
```

**Comment by noncom** [comment]
@aisamu Would you please do it if you can? Also, providing a way to specify a custom encoding would be great.

**Comment by aisamu** [comment]
Done! This is my first PR ever, pardon any faux-pas.

**Comment by michaelklishin** [comment]
Should be fixed in #7.
